frameworks/CSharp/aspnetcore-corert/PlatformBenchmarks/Utf8Json.Generated.cs

```diff
@@ -0,0 +1,163 @@
+﻿#pragma warning disable 618
```

Comment:
Add a comment here that this file was generated by the tool?

Reply:
Sure, done. But for the record, I'm not a huge fan of hand-editing generated files. I made sure the file name clearly states it was generated and that the commit description has the details of how.